src/components/chat/style.css

```diff
@@ -146,7 +146,28 @@
 }
 
 .chatright {
-  width: auto;
+
+  /* /////////////////////// */
+    align-items: center;
+    text-align: center;
```

Comment:
こまかくてすみませんが、textは
```suggestion
    text-align: left;
```
でも良いかも？と思いました！

centerの場合
![image](https://user-images.githubusercontent.com/111116632/194691913-33da6991-165b-445a-837a-88bb472811b9.png)

leftの場合
![image](https://user-images.githubusercontent.com/111116632/194691917-870a348c-f442-4087-a063-5a07e9415805.png)

Reply:
@kan-mayoshi 
遅れてすみません！左揃えに修正しました！
c8a556ec623adbc5ca75a7aecc125e507a790dd3